fares-data-build-tool/fdbt-site
# feat: updated site to use new env vars

# Description

-   This PR amends the site where service email addresses and phone numbers are displayed/used.

# Testing instructions

-   Pull down this branch and check that anywhere on the site where an email address is displayed is using an environment variable to get the value.
-   Also check that the site uses the correct email address from SES to send emails.

# Type of change


# Checklist:
